app/__init__.py

```diff
@@ -1,16 +1,8 @@
 from flask import Flask
 from flask_restful import Api
-from flask_jwt_extended import JWTManager
+from flask_cors import CORS
```

Comment:
why are you using CORS?

Reply:
To allow access by other domains  like GitHub pages